### DISCUSSION TITLE
Required Inside search of blog

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11145] [Q&A]
Need to implement the search see below:

![image](https://user-images.githubusercontent.com/5202291/152873509-f60c0f21-4014-4bb7-975b-ee056e48bc64.png)

Boundary is to search inside of that specific content type of Blog:

When anyone is search and it relate to blog it comes otherwise not? please advise

**Comment by Skrypt** [comment]
You need to use a custom Lucene Query that will return documents. For the autocomplete this will be a custom controller action that is invoked on the client-side with an AJAX call.

**Reply by saleemhaq** [reply]
> You need to use a custom Lucene Query that will return documents. For the autocomplete this will be a custom controller action that is invoked on the client-side with an AJAX call.

can you please share with me the example of custom Lucene Query and how i achieve from end to end please with ajax example.

**Reply by Skrypt** [reply]
I can't really do that as it would be as doing it for you ... I can't have examples for everything. Too much time-consuming.

**Reply by hishamco** [reply]
@saleemhaq as I told you many time, the team trying to help as much as they can but that doesn't mean to write everything on your behalf. Hope you invisit some time in docs and code and we are glad to help anyone

Thanks
